EIPS/eip-5489.md

```diff
@@ -89,7 +89,9 @@ interface IERC5489 {
     ) external;
 
     /**
-     * @dev
+     * @dev Throws if `tokenId` is not a valid NFT. URIs are defined in RFC 3986.
+     * The URI may point to a JSON file that confirms to the "EIP5489 Metadata JSON schema".
```

Comment:
Just to clarify, do you want to make the URI content to an optional compliant of EIP-5489 or do you want to make a  mandate?
```suggestion
     * The URI MUST point to a JSON file that confirms to the "EIP5489 Metadata JSON schema".
```